src/test-universal/check-size.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+IMAGE="$1"
+
+# Function to handle errors
+handle_error() {
+    local exit_code=$?
+    local line_number=$1
+    local command=$2
+    echo "Error occurred at line $line_number with exit code $exit_code in command $command"
+    exit $exit_code
+}
+trap 'handle_error $LINENO ${BASH_COMMAND%% *}' ERR
+echo "This is line $LINENO"
+# Check if bc is installed
+if ! command -v bc &> /dev/null; then
+    echo "bc is not installed. Installing..."
+    # Install bc using apt-get (for Debian-based systems)
+    sudo apt-get update
+    sudo apt-get install -y bc
+fi
+
+convert_gb_to_bytes() {
+    local gb="$1"
+    local bytes
+    bytes=$(echo "scale=0; $gb * 1024^3" | bc)
+    printf "%.0f\n" "$bytes"
+}
+CONTAINER_ID=$(docker ps -q --filter "label=test-container=$IMAGE")
+# Find the image ID of the container
+IMAGE_ID=$(docker inspect --format='{{.Image}}' "$CONTAINER_ID")
+# Find the size of the image
+IMAGE_SIZE=$(docker image inspect --format='{{.Size}}' "$IMAGE_ID")
+# Output the size
+echo "Size of the image $IMAGE_ID: $IMAGE_SIZE bytes"
+# Define the threshold size (in GB)
+gb_value=12.5
```

Comment:
```suggestion
gb_value=14
```

Currently, the size is 14 GB, see https://github.com/devcontainers/images/actions/runs/8146012516/job/22263557659#step:3:42668

Reply:
Done